### PR TITLE
Add Security Policy

### PR DESCRIPTION
**Pull request by gabibguti**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build was run locally and without warnings or errors
- [ ] All previous and new tests pass

## Pull request type

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming, typo fix)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Not having a Security Policy file.

Related Issue URL: Resolves https://github.com/tukaani-project/xz/issues/46

## What is the new behavior?

- Add Security Policy file

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

I've tried to keep the timelines of confirming a vulnerability report and fixing a vulnerability as open as possible. Let me know what you think and if this seems reasonable for maintainence.

### CODE REVIEW

**Comment by gabibguti** [comment]
I see you also have GitHub's Security Advisories feature enabled. So we could put 2 reporting options in the Security Policy, email and the [security advisory report link](https://github.com/tukaani-project/xz/security/advisories/new). Does it make sense?

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied only to the latest release.
```
Comment (JiaT75) [inline SECURITY.md:5]: This is true, that we will only apply the security updates when a new release comes out. However, we will also provide patches for security bugs on our project website (https://tukaani.org/xz), so this should be mentioned here. We will provide patches for all supported development branches, which will always be at least the current development branch and the current stable branch. Right now we still maintain an old stable branch, but that might not always be the case.

**Comment by gabibguti** [comment]
@JiaT75 Can you take a look to see if all your comments were addressed?

**Comment by JiaT75** [comment]
Looks great. Thanks for the contribution!

**Pull request merged**